samples/databases/wide-world-importers/wwi-ssdt/README.md

```diff
@@ -0,0 +1,98 @@
+
+### Publishing to SQL Server
+
+1. Open the solution **wwi-ssdt.sln** in Visual Studio. Skip this step if you have already opened the solution **wwi-sample.sln** in the root of this sample.
+
+2. Build the solution.
+
+3. Publish the WideWorldImporters database:
+    1. In the Solution Explorer, right-click the **WideWorldImporters** project, and select **Publish** to bring up the **Publish Database** dialog.
+    1. Click **Edit** to modify the **Target Database Connection** to point to your SQL Server 2016 (or later) instance.
+    1. Edit the **Database Name** to say "WideWorldImporters".
+    1. Click **Publish**.
+    1. Wait for publication to finish. You can monitor progress in the **Data Tools Operations** page in Visual Studio. In testing this took around 3 minutes.
+
+4. (Optional) Data population: After step 3, the database contains data for January 2013. This step populates data from February 2013 up to the current data.
+    1. Open SQL Server Management Studio, and connect to the WideWorldImporters database that was published in the previous step.
+    1. Execute the following script. This may take a while to complete - populating data from Feb 2013 to Jun 2016 took about 40 minutes in one test.
+
+```
+    EXEC DataLoadSimulation.PopulateDataToCurrentDate
+    @AverageNumberOfCustomerOrdersPerDay = 60,
+    @SaturdayPercentageOfNormalWorkDay = 25,
+    @SundayPercentageOfNormalWorkDay = 0,
+    @IsSilentMode = 0,
+    @AreDatesPrinted = 1;
+```
+
+To customize the period for data generation, leverage the stored procedure `DataLoadSimulation.DailyProcessToCreateHistory`.
+<br/><br/>The referenced stored procedure removes the temporal nature of the tables, and implements a series of triggers. It then emulates typical activities that would occur during each day. Finally, it removes the triggers and re-establishes the temporal tables. You can see the progress of the simulation in the Messages tab in SSMS as the query executes. (AreDatesPrinted controls whether dates are printed to the messages window as data is generated. IsSilentMode controls whether detailed output is printed. IsSilentMode = 1 produces just date output if AreDatesPrinted = 1.).
+Note that a different outcome is produced each time it is run as it uses many random values.
+StartDate and EndDate cover the period for generation. Other code populates the 2012 period when expanding the columnstore tables so do not populate back into 2012 or earlier with this procedure. The EndDate must also be at or before the current date as temporal tables do not allow future dates.
+You can configure the amount of data produced by modifying the number of orders per day. The default is 60 orders and produces a reasonable OLTP database size of around 93MB compressed. You are also able to configure how busy Saturday and Sunday are compared to normal Monday to Friday working days, as a percentage. The suggested values are 50% for Saturday and 0% for Sunday.
```

Comment:
"database size of around 93MB compressed" is this after just 60 orders?, in other words would it be correct to approximate that for a year [360 * 60 => 21600 * (X) MB = estimated size added to the database] This is particularly useful say when one wants to create a database of a particular size (5GB etc..)